helix-core/src/history.rs

```diff
@@ -277,37 +268,38 @@ fn parse_human_duration(s: &str) -> Result<Duration, String> {
             .to_string());
     }
 
-    let mut units = std::collections::HashMap::new();
+    let mut specified = [false; TIME_UNITS.len()];
```

Comment:
I like this.

Reply:
Me too, this ended up being a simplification, which is nice :)

Reply:
Wait no, probably I was tired that time when I review. I prefer it one usize but meh, this function is rarely used so I don't think it matters.